storage/deliveryjobrepo.go

```diff
@@ -115,6 +115,13 @@ func (djRepo *DeliveryJobDBRepository) MarkDeadJobAsInflight(deliveryJob *data.D
 	return err
 }
 
+func (djRepo *DeliveryJobDBRepository) DeleteJobsForMessage(message *data.Message) error {
+	err := transactionalWrites(djRepo.db, func(tx *sql.Tx) error {
+		return inTransactionExec(tx, emptyOps, "DELETE FROM job WHERE messageId like ?", args2SliceFnWrapper(message.ID), 0)
```

Comment:
Curious, why `like` and not `=`?

Reply:
No particular reason, just that I prefer using `LIKE` operator for string matching